blocks/test/fixtures/core__more.html

```diff
@@ -0,0 +1 @@
+<!-- wp:core/more {"text": "Continue reading", "noTeaser": true} /-->
```

Comment:
Edit: The issue was that we were checking that each block as a corresponding test file and this file contains `<!-- wp:{ name }` with a regex, I'm relying on the JSON file now to avoid recreating the parsing mechanism in the test